scripts/install-cni.sh

```diff
@@ -37,19 +37,20 @@ set -u -e
 # Get CNI spec template if needed.
 if [ "${ENABLE_CALICO_NETWORK_POLICY}" == "true" ]; then
   echo "Calico Network Policy is enabled"
-  if [ -z "${CALICO_CNI_SPEC_TEMPLATE_FILE:-}" ]; then
-    echo "No Calico CNI spec template is specified. Exiting (0)..."
+  # Backwards compatibility support
+  if [ -n "${CALICO_CNI_SPEC_TEMPLATE_FILE}" ] && [ -n "${CALICO_CNI_SPEC_TEMPLATE}" ]; then
+    # Use environment overrides
+    calico_cni_file="${CALICO_CNI_SPEC_TEMPLATE_FILE}"
+    cni_spec="${CALICO_CNI_SPEC_TEMPLATE}"
+  elif [ "${OVERWRITE_CALICO_CNI_FILE:-"false"}" == "false" ]; then
```

Comment:
This skip all the iptables rules, is that the intention?

Reply:
Yes, in the original logic, if `CALICO_CNI_SPEC_TEMPLATE_FILE` and `CALICO_CNI_SPEC_TEMPLATE` were not set, then the script exits.